lib/hawkular/operations/operations_api.rb

```diff
@@ -53,14 +53,29 @@ def to_msg_hash
     # @example
     #   Hawkular::Operations::OperationsClient.new(credentials: {username: 'jdoe', password: 'password'})
     def initialize(args)
-      args[:host] ||= 'localhost:8080'
+      ep = args[:entrypoint]
```

Comment:
btw I'm confused where is entrypoint coming from? I read lines 44-51 and don't see it there